schemas/bridge/SCHEMA.md

```diff
@@ -0,0 +1,135 @@
+| chain_id                 | The standard chain ID where the bridger resides.          | number |
+| bridge_type              | The type of bridge (pool-based, mint-based, or intent-based). | string |
+
+### Pool Snapshot
+
+Pool level snapshots for pool and mint based bridges.
+
+| Property                | Description                                               | Type   |
+|-------------------------|-----------------------------------------------------------|--------|
+| timestamp                | The timestamp of the snapshot.                            | number |
+| pool_address             | The contract address of the pool.                         | string |
+| token_address            | The contract address of the token being bridged or pooled. | string |
+| token_index              | (Optional, only applicable to pool-based) The index of the pooled token in the smart contract. | number |
+| token_symbol             | The symbol of the token.                                  | string |
+| token_amount             | The amount of the supplied token (in a pool-based bridge), decimal normalized. | number |
+| token_amount_usd         | The amount supplied, in USD.                              | number |
+| volume_usd               | The bridged volume in USD, during this snapshot.          | number |
+| fees_usd                 | The fees collected in USD, during the snapshot period.    | number |
+| chain_id                 | The standard chain ID where the pool resides.             | number |
+| bridge_type              | The type of bridge (pool-based, mint-based, or intent-based). | string |
+
+### Bridge Transactions
+
+List of bridge transactions.
+
+| Property                | Description                                               | Type   |
+|-------------------------|-----------------------------------------------------------|--------|
+| timestamp                | The timestamp of the transaction.                         | number |
+| block_number             | The block number of the transaction.                      | number |
+| transaction_hash         | The hash of the transaction this bridge originated from.  | string |
+| source_chain_id          | The source standard chain ID.                             | number |
+| destination_chain_id     | The destination standard chain ID.                        | number |
+| volume_usd               | The volume of the transaction in USD.                     | number |
+| user_address             | The address of the user initiating the transaction.       | string |
+| solver_address           | The address of the solver (for intent-based bridges).     | string |
+| pool_address             | The address of the pool (for pool-based and mint-based bridges). | string |
+| bridge_type              | The type of bridge (pool-based, mint-based, or intent-based). | string |
```

Comment:
lets add in a few more columns

`log_index`
`token_address`
`token_amount`

maybe `volume_usd` to `token_amount_usd`

Reply:
are we doing `transaction_hash` or transaction_id`?

Reply:
I've been using `transaction_hash` elsewhere. There is actually an update to nft/dex ag to that, will submit that.